src/main/webapp/WEB-INF/jsp/submitFeedback.jsp

```diff
@@ -20,91 +20,147 @@
 --%>
 <jsp:directive.include file="/WEB-INF/jsp/include.jsp"/>
 
+<link href="<c:url value="/css/feedback.css"/>" type="text/css" rel="stylesheet"/>
+
 <script src="<rs:resourceURL value="/rs/jquery/1.11.0/jquery-1.11.0.min.js"/>" type="text/javascript"></script>
+<script src="/ResourceServingWebapp/rs/jquery-migrate/1.2.1/jquery-migrate-1.2.1.min.js" type="text/javascript"></script>
```

Comment:
Hi @timlevett 
It's a link to the ressourceserver, you can use a link to jQuery's CDN, if you want.
But if it is for myUW angularJS front, because it don't have a multi columns layout dashboard, you can comment these lines:
in viewFeedback.jsp:
from
https://github.com/cousquer/FeedbackPortlet/blob/42c3b4ed2835ebb038fafad618b1a1489d650089/src/main/webapp/WEB-INF/jsp/viewFeedback.jsp#L77
to
https://github.com/cousquer/FeedbackPortlet/blob/42c3b4ed2835ebb038fafad618b1a1489d650089/src/main/webapp/WEB-INF/jsp/viewFeedback.jsp#L131

in theme.less (beware of the comment...):
from
https://github.com/cousquer/FeedbackPortlet/blob/42c3b4ed2835ebb038fafad618b1a1489d650089/src/main/webapp/less/theme.less#L74
to
https://github.com/cousquer/FeedbackPortlet/blob/42c3b4ed2835ebb038fafad618b1a1489d650089/src/main/webapp/less/theme.less#L88